web/src/components/planner/StatisticsPower.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div class="px-2">
+    <h1 class="text-h5 mb-4">
+      <i class="fas fa-power-off mr-3" />Power Consumption and Generation
+    </h1>
+    <p v-show="helpText" class="mb-4">
+      <i class="fas fa-info-circle mr-2" />Shows world level power consumption and generation data.
+    </p>
+    <v-chip
+      class="sf-chip yellow"
+      variant="tonal"
+    >
+      <i class="fas fa-bolt mr-2" />{{ formatNumber(totalPower.totalPowerConsumed) }} MW consumed
+    </v-chip>
+    <v-chip
+      class="sf-chip green"
+      variant="tonal"
+    >
+      <i class="fas fa-solar-panel mr-2" />{{ formatNumber(totalPower.totalPowerProduced) }} MW generated
+    </v-chip>
+  </div>
```

Comment:
Would be nice to have a chip showing the difference between the two, and if it's in deficit show up in red